Jenkinsfile

```diff
@@ -1,25 +1,19 @@
-#!groovy
+#!groovyF
```

Comment:
Is this a typo? I cant find any reference to `groovyF`

Reply:
Typo